isamples_metadata/Transformer.py

```diff
@@ -22,20 +22,18 @@ def transform(self) -> typing.Dict:
             The provider record transformed into an iSamples record
         """
         transformed_record = {
-            "$schema": "../iSamplesSchemaBasicSMR.json",
-            "@id": "https://data.isamples.org/digitalsample/{0}/{1}".format(
-                self.sample_identifier_scheme(), self.sample_identifier_value()
-            ),
+            "$schema": "../../iSamplesSchemaBasic0.2.json",
+            "@id": self.id_string(),
             "label": self.sample_label(),
-            "sampleidentifier": "{0}:{1}".format(
-                self.sample_identifier_scheme(), self.sample_identifier_value()
-            ),
+            "sampleidentifier": self.sample_identifier_string(),
             "description": self.sample_description(),
             "hasContextCategory": self.has_context_categories(),
             "hasMaterialCategory": self.has_material_categories(),
             "hasSpecimenCategory": self.has_specimen_categories(),
+            "informalClassification": self.informal_classification(),
             "keywords": self.keywords(),
             "producedBy": {
+                "@id": self.produced_by_id_string(),
```

Comment:
New field we'll need to ingest in solr.

Reply:
@datadavev please confirm

Reply:
Actually, though, now that I think about it I wonder if this is was already taken are of by the previous relations solr work.

Reply:
Lets add as a new field to the search index.